src/app/components/DynamicProgress.js

```diff
@@ -4,37 +4,49 @@ import { c } from 'ttag';
 import { Icon } from 'react-components';
 import contactLoader from 'design-system/assets/img/pm-images/contact-export-loader.gif';
 
-const DynamicProgress = ({ id, alt, displayDuring, displayEnd, value, max, endPostponed, ...rest }) => {
+const DynamicProgress = ({
+    id,
+    alt,
+    displayDuring = c('Progress bar description').t`Progress`,
+    displaySuccess = c('Progress bar description').t`Completed`,
+    displayFailed = c('Progress bar description').t`Failed`,
+    value,
+    max = 100,
+    endPostponed = false,
+    failed = false,
+    ...rest
+}) => {
+    const loading = value < max || endPostponed;
+    const icon = loading ? (
+        <img src={contactLoader} alt={`${alt}-loader`} />
```

Comment:
Use `<Loader />` to be consistent with other places.

Reply:
I thought for contacts we had this specific gif `contactLoader`, which is bigger than the one in `Loader` and which would just show up on export, import and merge. [This design](https://app.zeplin.io/project/5bb1deae290afc111d140a3f/screen/5cdbf978c2475c1e2bc30668) has been overridden then?